src/auth/screens/events.screen.js

```diff
@@ -86,8 +89,11 @@ class Events extends Component {
     }
   }
 
-  getUserEvents = (user = this.props.user) => {
+  getUserEvents = () => {
```

Comment:
We're passing a user into this function on lines 82 and 88, and this would be therefore ignoring it. Maybe something like this will work?

```javascript
getUserEvents = ({ user, accessToken } = this.props) => {
    // ...
}
```

While changing lines 82 and 88 to pass in `this.props` and `nextProps` respectively.

Reply:
I tried to do it, it did not work.

Reply:
Agree with @andrewda. Line 82 shouldn't matter since it's already defaulting to `this.props.user` (would prefer if you can remove the argument call in there (line 82) if you can too :)

But if you're removing the arguments entirely from the function declaration in line 92, line 88 doesn't look it would work at all. We're passing in `nextProps.user` but in here it's not being used at all..

Curious why 

```
getUserEvents = ({ user, accessToken } = this.props) => {
    // ...
}
```

doesn't work? Destructuring and applying a default should work if I'm not mistaken